### PR TITLE
Kotlin asinh

### PR DESCRIPTION
**Pull request by Anudhyan**
### Description
-Added new math method asinh() entry for the Kotlin programming language
-Worked for the first time with Codecademy
-It's a term entry

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Anudhyan** [comment]
@SSwiniarski sir please check my pull request and kindly approve it

File: content/kotlin/concepts/math-methods/terms/asinh/asinh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/asinh/asinh.md:3]:
```suggestion
Description: 'Returns the inverse hyperbolic sine of a numeric argument.'
```

File: content/kotlin/concepts/math-methods/terms/asinh/asinh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/asinh/asinh.md:30]:
```suggestion
```
Delete extra blank line.

File: content/kotlin/concepts/math-methods/terms/asinh/asinh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/asinh/asinh.md:32]:
```suggestion
The example below is finding the value of the inverse hyperbolic sine of 1.0
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Anudhyan I did a first review and had a couple of comments.

File: content/kotlin/concepts/math-methods/terms/asinh/asinh.md
Comment (Anudhyan) [reply content/kotlin/concepts/math-methods/terms/asinh/asinh.md:30]: Kindly check. I have fixed it, sir !

**Comment by Anudhyan** [comment]
@SSwiniarski sir please check it soon and approve my first ever pull request !

**Review by SSwiniarski — APPROVED** [review]
@Anudhyan Passing this on to a second review.

**Comment by Anudhyan** [comment]
> @Anudhyan Passing this on to a second review.

Then what will I have to do sir , in the mean time?

**Comment by SSwiniarski** [comment]
> > @Anudhyan Passing this on to a second review.
> 
> Then what will I have to do in the mean time ,sir? @SSwiniarski

@Anudhyan you don't have to do anything unless the second review comes up with more changes.

**Review by caupolicandiaz — APPROVED** [review]
@Anudhyan @SSwiniarski Just made a few small direct edits. This is good to merge

**Pull request merged**
